Makefile

```diff
@@ -41,9 +43,27 @@ else
 	docker pull quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG)
 endif
 
+.PHONY: ma-builder
+ma-builder:
+ifdef BUILD_BUILDER_IMAGE
+	docker buildx build --push --platform ${ARCH} \
```

Comment:
Related to my previous comment, seems like if we were to remove the `--push` option here we could keep a single target instead of duplicating everything. Also, would you mind elaborating on why we need to use buildx? Why can't we keep the regular `docker build` and let people choose what they want to use for building with the `DOCKER_BUILDKIT` env variable?

Reply:
Could you please provide an example of how that can be achieved? Because as far as I'm aware, docker build will only create a platform specific image, the one of the host.